COPYRIGHT

```diff
@@ -0,0 +1,21 @@
+Except as otherwise noted (below and/or in individual files), this project is
+licensed under the Apache License, Version 2.0 <LICENSE-APACHE>
+<http://www.apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT>
+<http://opensource.org/licenses/MIT>, at your option.
+
+Copyright is retained by contributors and/or the organisations they
+represent(ed) -- this project does not require copyright assignment. Please see
+version control history for a full list of contributors. Note that some files
+may include explicit copyright and/or licensing notices.
+
+The following contributors wish to explicitly make it known that the copyright
+of their contributions is retained by an organisation:
+
+    Edd Barrett <vext01@gmail.com>: copyright retained by
+      King's College London
+    Jacob Hughes <jh@jakehughes.uk>: copyright retained by
+      King's College London
+    Sarah Mount <sarah.mount@kcl.ac.uk>: copyright retained by
+      King's College London
+    Laurence Tratt <laurie@tratt.net>: copyright retained by
+      King's College London
```

Comment:
I'm missing again. And I don't think Edd or Jake have any contributions in this repo, unless you want to put them there for future proofing.

Reply:
Oops, you're quite right! Fixing...

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/118/commits/360f87e816f5ff4acf7194bc9ce70aabdfa58037